x-pack/plugins/security_solution/public/entity_analytics/components/asset_criticality_file_uploader/components/file_picker_step.tsx

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiCodeBlock,
+  EuiFilePicker,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  useEuiTheme,
+} from '@elastic/eui';
+import { css } from '@emotion/css';
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import { euiThemeVars } from '@kbn/ui-theme';
+import {
+  CRITICALITY_CSV_MAX_SIZE_BYTES,
+  ValidCriticalityLevels,
+} from '../../../../../common/entity_analytics/asset_criticality';
+import { useFormatBytes } from '../../../../common/components/formatted_bytes';
+import { SUPPORTED_FILE_EXTENSIONS, SUPPORTED_FILE_TYPES } from '../constants';
+
+interface AssetCriticalityFilePickerStepProps {
+  onFileChange: (fileList: FileList | null) => void;
+  isLoading: boolean;
+  errorMessage?: string;
+}
+
+const sampleCSVContent = `user,user-001,low_impact\nuser,user-002,medium_impact\nhost,host-001,extreme_impact`;
+
+const listStyle = css`
+  list-style-type: disc;
+  margin-bottom: ${euiThemeVars.euiSizeL};
+  margin-left: ${euiThemeVars.euiSizeL};
+  line-height: ${euiThemeVars.euiLineHeight};
+`;
+
+export const AssetCriticalityFilePickerStep: React.FC<AssetCriticalityFilePickerStepProps> =
+  React.memo(({ onFileChange, errorMessage, isLoading }) => {
+    const formatBytes = useFormatBytes();
+    const { euiTheme } = useEuiTheme();
+    return (
+      <>
+        <EuiSpacer size="m" />
+        <EuiPanel color={'subdued'} paddingSize="l" grow={false} hasShadow={false}>
+          <EuiTitle size="xxs">
+            <h3>
+              <FormattedMessage
+                defaultMessage="CSV File Format Requirements"
+                id="xpack.securitySolution.entityAnalytics.assetCriticalityUploadPage.csvFileFormatRequirements"
+              />
+            </h3>
+          </EuiTitle>
+          <EuiSpacer size="s" />
+
+          <ul className={listStyle}>
```

Comment:
I thought we should use `styled` for scenarios like this?

Reply:
I couldn't find good documentation for it, but there is the EUI team ticket https://github.com/elastic/eui/issues/3912